Jenkinsfile

```diff
@@ -1,104 +1,32 @@
 #!groovy
-@Library('waluigi@v5.0.0') _
-
-standardProperties()
-
-def withPublishCredentials(String dirPath, cl) {
-  withCredentials([
-    string(credentialsId: 'npm_live_token', variable: 'NPM_LIVE_TOKEN')
-  ]) {
-    String filePath = "${dirPath}/.npmrc"
-    try {
-      sh """
-      cat > "${filePath}" <<EOF
-      //registry.npmjs.org/:_authToken=${env.NPM_LIVE_TOKEN}
-      EOF
-      """.stripIndent()
-      sh """
-      cd "${dirPath}" && \
-      npm whoami --registry https://registry.npmjs.org/
-      """.stripIndent()
-      cl()
-    } finally {
-      sh "rm \"${filePath}\""
-    }
-  }
-}
-
-timestamps {
-  podTemplate(
-    cloud: "builds",
-    yaml: '''
-    apiVersion: "v1"
-    kind: "Pod"
-    spec:
-      securityContext:
-        runAsUser: 1000
-        runAsGroup: 1000
-      containers:
-      - name: "node"
-        image: "public.ecr.aws/docker/library/node:18"
-        command:
-        - "sleep"
-        args:
-        - "infinity"
-        imagePullPolicy: "Always"
-        resources:
-          requests:
-            memory: "4Gi"
-            cpu: "500m"
-          limits:
-            memory: "4Gi"
-            cpu: "4"
-    '''.stripIndent()
-  ) {
-    node(POD_LABEL) {
-      container('jnlp') {
-        stage("checkout") {
-          checkout localBranch(scm)
-        }
-      }
-      container('node') {
-        stage("dependencies") {
-          sh 'yarn config set registry https://registry.npmjs.org/'
-          yarnInstall()
-        }
-
-        stage("stamp") {
-          sh "yarn beehive-flow stamp"
-        }
-
-        stage("build") {
-          sh "yarn build"
-        }
-
-        stage("lint") {
-          sh "yarn lint"
-        }
-
-        bedrockBrowsers(testDirs: [ "tinymce-angular-component/src/test/ts/browser" ])
-
-        stage("storybook") {
-          def status = beehiveFlowStatus();
-          if (status.branchState == 'releaseReady' && status.isLatest) {
-            sshagent (credentials: ['ccde5b3d-cf13-4d70-88cf-ae1e6dfd4ef4']) {
-              sh 'yarn storybook-to-ghpages'
-            }
-          } else {
-            echo "Skipping publish as is not latest release"
-            sh 'yarn storybook-to-ghpages --dry-run'
-          }
-        }
-
-        stage("publish") {
-          sshagent(credentials: ['jenkins2-github']) {
-            withPublishCredentials('dist/tinymce-angular') {
-              sh "yarn beehive-flow publish --working-dir dist/tinymce-angular"
-            }
-            sh "yarn beehive-flow advance-ci"
+@Library('waluigi@feature/DEVOPS-449') _
```

Comment:
This will need changing to something more stable before we merge this, probably the `feature/containers` branch instead since we're not quite ready to release fully yet.

- [ ] Swap to more stable waluigi tag or branch